### PR TITLE
PackageTests.md: explain how to restrict networking

### PR DESCRIPTION
**Pull request by cpaelzer**
Triggered by +1 maintenance by Paride, I hope this helps to recreate such cases.

### CODE REVIEW

File: PackageTests.md [hunk 1]
```diff
@@ -169,6 +169,22 @@ $ autopkgtest --no-built-binaries --apt-upgrade --setup-commands setup-testbed -
...
+it often is worth to retry a local VM based repro with this to check if it
+fails this way.
+
+What you need to do is adding the internal proxy (needs VPN up); or if you want
+to try another proxy of your choice that is rather restrictive. Then add this
+to the call of autopkgtest `--env='no_proxy=127.0.0.1,127.0.1.1,localhost,localdomain,novalocal,internal,archive.ubuntu.com,security.ubuntu.com,ddebs.ubuntu.com,changelogs.ubuntu.com,ppa.launchpad.net' --env='http_proxy=http://squid.internal:3128'`
+
+Example when we tracked down an issue with ulfius:
+```
+$ sudo autopkgtest --env='no_proxy=127.0.0.1,127.0.1.1,localhost,localdomain,novalocal,internal,archive.ubuntu.com,security.ubuntu.com,ddebs.ubuntu.com,changelogs.ubuntu.com,ppa.launchpad.net' --env='http_proxy=http://squid.internal:3128' --no-built-binaries --apt-upgrade --apt-pocket=proposed=src:qemu --shell-fail ulfius_2.7.1-3.dsc  -- qemu --qemu-options='-cpu host' --ram-size=1536 --cpus 2 ~/work/autopkgtest-impish-amd64.img
```
Comment (paride) [inline PackageTests.md:186]: Just a couple of questions on this line:
1. Why `sudo`? I normally run autopkgtest-virt-qemu autopkgtests without sudo and they work fine.
2. I'd make the example line more minimal, so the attention goes to the relevant part for the example. That is I'd drop `--apt-pocket=proposed=src:qemu` and maybe also `--qemu-options='-cpu host' --ram-size=1536 --cpus 2`, unless there's a good reason to keep them and that I'm not seeing, of course!

**Review by paride — COMMENTED** [review]
Super helpful, thanks! Just a couple of inline comments.

File: PackageTests.md (hunk 1 above)
Comment (cpaelzer) [reply PackageTests.md:186]: sudo because of image ownership fluctuating in my case, I'll drop.

And yes I can make it simpler to focus on the new bit.

Comment (cpaelzer) [reply PackageTests.md]: I also run autopkgtest from git usually, but that doesn't matter.

Done, should be good now.

**Review by paride — APPROVED** [review]
Thanks!

**Pull request merged**